src/main/java/org/cryptomator/common/networking/SSLContextWithPKCS12File.java

```diff
@@ -0,0 +1,31 @@
+package org.cryptomator.common.networking;
+
+import org.cryptomator.integrations.common.CheckAvailability;
+import org.cryptomator.integrations.common.OperatingSystem;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.CertificateException;
+
+/**
+ * SSLContextProvider for Linux using a PKCS#12 file as trust store
+ */
+@OperatingSystem(OperatingSystem.Value.LINUX)
+public class SSLContextWithPKCS12File extends SSLContextDifferentTrustStoreBase implements SSLContextProvider {
+
+	private static final String CERT_FILE_LOCATION_PROPERTY = "org.cryptomator.common.integrations.sslcontext.pkcs12file";
```

Comment:
adjust property name

Reply:
Done in https://github.com/cryptomator/cryptomator/pull/3675/commits/66a0cfe7dd6d495887dc029ee63baaede65dc335